docs/pages/admin-guides/access-controls/access-request-plugins/ssh-approval-jira.mdx

```diff
@@ -166,7 +166,7 @@ enter the domain name you created for the plugin earlier, plus port `8081`.
 </TabItem>
 <TabItem label="Helm Chart">
 
-Enter "Teleport Access Request Plugin" in the "Name" field. In the "URL" field,
+Enter "Access Request Plugin" in the "Name" field. In the "URL" field,
```

Comment:
I would keep "Teleport" in the name, so that when the reader navigates to Jira and sees the list of webhooks they will know that this one is for Teleport's integration.

Plus, the webhook name we advised on line 163 is "Teleport Access Request Plugin" so we definitely want this to match.